### ISSUE TITLE
Log unhandled events

### ISSUE THREAD

**Issue by gbirke** [rvagg/github-webhook#7]
It would be nice if an event where no rules matched would create a log entry.

**Comment by imontoyou89** [comment]
[liar89-master.zip](https://github.com/rvagg/github-webhook/files/3775862/liar89-master.zip)

**Comment by rvagg** [comment]
@gbirke yes, that might be nice, would you mind preparing a pull request to add this? As I don't personally need it, it's probably not going to be added by me.
